main.py

```diff
@@ -911,7 +924,11 @@ def _load_evc(self, circuit_dict):
                 f"Could not load EVC: dict={circuit_dict} error={exception}"
             )
             return None
-
+        except KytosTagError as exception:
```

Comment:
Trivial detail but you can group exception types since you're using the same code in the except claus, you can turn the prior except as `except (ValueError, KytosTagError) as exception:`, that way eliminating the need to repeat the `log.error` and return None statement

Reply:
Got it, grouped similar cases. Commit [918e21324fdd2dc3d6615cf967613ae75802fb31](https://github.com/kytos-ng/mef_eline/pull/407/commits/918e21324fdd2dc3d6615cf967613ae75802fb31)